src/app/api/webhook/route.ts

```
@@ -82,5 +84,62 @@ export async function POST(req: NextRequest) {
+            
+    const event = payload as CallEndedEvent;
+        const meetingId = event.call.custom?.meetingId;
+
+        if (!meetingId) {
+            return NextResponse.json({ error: "Missing meetingid" }, { status: 400 })
+        }
+
+        await db.update(meetings).set({ status: "processing", endedAt: new Date() }).where(and(eq(meetings.id, meetingId), eq(meetings.status, "active")));
+
+
+    }else if (eventType === "call.transcription_ready") {
```

Biome (2.1.2)

[error] This branch can never execute. Its condition is a duplicate or covered by previous conditions in the if-else-if chain.



(lint/suspicious/noDuplicateElseIf)